src/NoteWorkspace.ts

```diff
@@ -383,6 +394,84 @@ export class NoteWorkspace {
     );
   }
 
+  static newNoteFromSelection(context: vscode.ExtensionContext) {
+    const originEditor = vscode.window.activeTextEditor;
+
+    if (!originEditor) {
+      // console.debug('Abort: no active editor');
+      return;
+    }
+    const { selection } = originEditor;
+    const noteContents = originEditor.document.getText(selection);
```

Comment:
Is this resilient to the case where there are multiple instances of the same text in the origin? My read of `Selection` in vscode API is yes, but just wanted to check w you